modules/@ergonode/media/src/extends/ui/components/Grid/Layout/Table/Cells/Edit/GridFileEditCell.vue

```diff
@@ -49,17 +47,11 @@ export default {
         },
     },
     computed: {
-        positionStyle() {
-            const {
-                x,
-                y,
-            } = this.bounds;
-
-            return {
-                top: `${y}px`,
-                left: `${x}px`,
-                width: '304px',
-            };
+        height() {
+            return 181;
+        },
+        width() {
+            return 304;
         },
```

Comment:
these values are repeated in many components, maybe they should be moved to DEFAULTS ?